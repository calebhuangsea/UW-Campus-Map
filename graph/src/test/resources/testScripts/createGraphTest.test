# createGraph testing file
# test that a graph can be created
# test ListNodes method for empty graph

# create a graph
CreateGraph graph1

# print nodes in the graph
ListNodes graph1