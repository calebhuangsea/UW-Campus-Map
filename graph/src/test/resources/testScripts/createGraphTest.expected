# createGraph testing file
# test that a graph can be created
# test ListNodes method for empty graph

# create a graph
created graph graph1

# print nodes in the graph
graph1 contains: